rita_extender/src/lib.rs

```diff
@@ -0,0 +1,212 @@
+#[macro_use]
+extern crate log;
+
+#[macro_use]
+extern crate lazy_static;
+
+mod error;
+
+use actix_async::System as AsyncSystem;
+use error::RitaExtenderError;
+use rita_client::dashboard::wifi::WifiInterface;
+use rita_client::extender::get_device_mac;
+use rita_client::extender::ExtenderCheckin;
+use rita_client::get_wifi_config_internal;
+use rita_client::set_ssid;
+use rita_client::WifiSsid;
+
+use std::sync::Arc;
+use std::sync::RwLock;
+use std::thread;
+use std::time::Duration;
+use std::time::Instant;
+
+use rita_client::extender::ExtenderUpdate;
+
+const EXTENDER_CHECKIN_TIMEOUT: Duration = Duration::from_secs(5);
+const EXTENDER_LOOP_SPEED: Duration = Duration::from_secs(30);
+pub const DEFAULT_UPSTREAM_ENDPOINT: &str = "http://192.168.10.1:4877/extender_checkin";
+
+lazy_static! {
+    /// A list of settings used to interally set /etc/config/wireless, and to sent up to main router.
+    /// Stored here in memory to check if settings changed before manually running the commands to modify them
+    pub static ref EXTENDER_SETTINGS: Arc<RwLock<ExtenderCheckin>> = Arc::new(RwLock::new(ExtenderCheckin::default()));
```

Comment:
Currently this storage doesnt do anything since I apply the settings I get from the main router every tick, without checking if the settings changed. Wanted to get your opinion if this was ok before I removed this

Reply:
if that's the case go ahead and remove these.